.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 node_modules
-build/
\ No newline at end of file
+build/
+honeyswap-xdai.tokenlist.json
```

Comment:
Why ignore it if it's going to be deleted from the repo?

Reply:
I remove it in favor of #13.